test/quantization/fx/test_equalize_fx.py

```diff
@@ -110,3 +124,142 @@ def test_input_weight_observer(self, input_qdtype, input_qscheme, weight_qdtype,
             ref_scales, dtype=weight_qparams[0].dtype), atol=0.0001))
         self.assertTrue(torch.allclose(weight_qparams[1], torch.tensor(
             ref_zero_points, dtype=weight_qparams[1].dtype), atol=1))
+
```

Comment:
General comment: For all these tests seems like just the model type and node_occurance is changing. Thoughts on consolidating it into one/two tests and looping over the different types?

Reply:
Ah, that would be a more concise way -- would this be structured as looping over a list of (model type, node_occurance) tuples?

Reply:
yes, I think there are a few tests in test_quantize_fx that are already doing this.